frontend_vue/src/components/constants.ts

```diff
@@ -13,7 +13,6 @@ export const TOKENS_TYPE = {
   WEB_BUG: 'web',
   DNS: 'dns',
   LOG4SHELL: 'log4shell',
-  CLONED_WEBSITE: 'clonedsite',
```

Comment:
any reason we removing this?

Reply:
Just because this hasn't been used anywhere so far 
( I added it as a sample at the beginning, clonedsite Token is not integrated yet )